benchmark/events/ee-emit-mixed.js

```diff
@@ -0,0 +1,23 @@
+var common = require('../common.js');
+var EventEmitter = require('events').EventEmitter;
+
+var bench = common.createBenchmark(main, {n: [2e6]});
+
+function main(conf) {
+  var n = conf.n | 0;
+
+  var ee = new EventEmitter();
+
+  for (var k = 0; k < 10; k += 1) {
+    if (k % 2)
+      ee.once('dummy', function() {});
+    else
+      ee.on('dummy', function() {});
+  }
+
+  bench.start();
+  for (var i = 0; i < n; i += 1) {
+    ee.emit('dummy');
```

Comment:
is this a very fair benchmark? after all, wouldn't any run after the first run be only to the `.on()` listeners?